File: extract.js
import sharp from 'sharp';

async function processImage() {
    try {
        const image = sharp('Assets/Rehab Process.jpeg');
        const metadata = await image.metadata();
        const { width, height } = metadata;
        
        const { data, info } = await image.raw().ensureAlpha().toBuffer({ resolveWithObject: true });
        
        for (let i = 0; i < data.length; i += 4) {
            const r = data[i];
            const g = data[i+1];
            const b = data[i+2];
            
            if (r > 200 && g > 200 && b > 200) {
                data[i] = 255;
                data[i+1] = 255;
                data[i+2] = 255;
                data[i+3] = 255;
            } else {
                data[i+3] = 0;
            }
        }
        
        await sharp(data, { raw: { width, height, channels: 4 } })
            .trim()
            .webp({ quality: 100 })
            .toFile('Assets/Rehab Process Transparent.webp');
            
        console.log("Success with node");
    } catch(err) {
        console.error("Error", err);
    }
}
processImage();

File: extract.py
import cv2
import numpy as np
import sys

try:
    img = cv2.imread('Assets/Rehab Process.jpeg')
    if img is not None:
        gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
        _, mask = cv2.threshold(gray, 200, 255, cv2.THRESH_BINARY)
        h, w = img.shape[:2]
        result = np.zeros((h, w, 4), dtype=np.uint8)
        result[mask == 255] = [255, 255, 255, 255]
        cv2.imwrite('Assets/Rehab Process Transparent.webp', result)
        print("Success OpenCV")
        sys.exit(0)
except Exception as e:
    pass

try:
    from PIL import Image
    img = Image.open('Assets/Rehab Process.jpeg').convert("RGBA")
    data = img.getdata()
    new_data = []
    for item in data:
        if item[0] > 200 and item[1] > 200 and item[2] > 200:
            new_data.append((255, 255, 255, 255))
        else:
            new_data.append((255, 255, 255, 0))
    img.putdata(new_data)
    
    # Check bounds to trim if necessary
    bbox = img.getbbox()
    if bbox:
        img = img.crop(bbox)
        
    img.save('Assets/Rehab Process Transparent.webp', 'WEBP')
    print("Success Pillow")
except Exception as e:
    print("Error:", e)
